README.md

```diff
@@ -133,13 +133,14 @@ in-memory hash table.
 The work of wait event statistics collector worker is controlled by following
 GUCs.
 
-|         Parameter name              | Data type |                  Description                | Default value |
-| ----------------------------------- | --------- | ------------------------------------------- | ------------: |
-| pg_wait_sampling.history_size       | int4      | Size of history in-memory ring buffer       |          5000 |
-| pg_wait_sampling.history_period     | int4      | Period for history sampling in milliseconds |            10 |
-| pg_wait_sampling.profile_period     | int4      | Period for profile sampling in milliseconds |            10 |
-| pg_wait_sampling.profile_pid        | bool      | Whether profile should be per pid           |          true |
-| pg_wait_sampling.profile_queries    | bool      | Whether profile should be per query			|          true |
+| Parameter name                   | Data type | Description                                 | Default value |
+|----------------------------------| --------- |---------------------------------------------|--------------:|
+| pg_wait_sampling.history_size    | int4      | Size of history in-memory ring buffer       |          5000 |
+| pg_wait_sampling.history_period  | int4      | Period for history sampling in milliseconds |            10 |
+| pg_wait_sampling.profile_period  | int4      | Period for profile sampling in milliseconds |            10 |
+| pg_wait_sampling.profile_pid     | bool      | Whether profile should be per pid           |          true |
+| pg_wait_sampling.profile_queries | bool      | Whether profile should be per query         |          true |
+| pg_wait_sampling.sample_cpu      | bool      | Whether on CPU backends should be sampled   |         false |
```

Comment:
Not sure about naming, but couldn't come up with anything better.

Reply:
I'm not perfectly happy with it either, but like you that's the best I could come up with. People should want these events because it counts processes that are either running on CPU or waiting to do so, so I wanted the name to at least point them in the right direction. I would argue also having it on by default, as I expect most people to just be using the profile functionality and it makes the output of that much easier to interpret.

Reply:
OK, you convinced me, let's turn it on by default. Otherwise PR is ready for merge. Thank you.